README.md

```diff
@@ -1,30 +1,27 @@
 ![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
 
-# MODULE_NAME Service
+# <LIBRARY_NAME> Library
 
 *TODO after cloning:*
-
-1. *Replace the repository name for each:*
+1. Replace the repository name on these badges:
```

Comment:
If we italicize in svc, why getting rid of it here?

Reply:
Fixed in https://github.com/Arrow-air/template-rust-lib/pull/1/commits/10f99ba487d694fce868138e25b71987138371f9